### ISSUE TITLE
Incorrect aria-label is defined for 'Maximize Sidebar' button as "Minimize Sidebar" button.

### ISSUE THREAD

**Issue by ElinorW** [microsoftgraph/microsoft-graph-explorer-v4#841]
**Describe the bug**
Incorrect aria-label is defined for 'Maximize Sidebar' button as "Minimize Sidebar" button. After activating 'Minimize Sidebar' button and then again navigating back to this button, screen reader conveys 'Minimize Sidebar' button.

**To Reproduce**
1. Open the URL in Anaheim edge dev browser.
2. Navigate to 'Minimize Sidebar' button and activate it. Sidebar navigation pane gets collapsed.
3. Navigate to 'Maximize Sidebar' button and verify its name using F12 developer tool.
4. Observe the issue.

**Expected behavior**
Correct aria-label should be defined for 'Maximize Sidebar' button as "Maximize Sidebar" button. After activating 'Minimize Sidebar' button and then again navigating back to this button, the screen reader should convey like 'Maximize Sidebar' button.
 
[AB#8112](https://microsoftgraph.visualstudio.com/0985d294-5762-4bc2-a565-161ef349ca3e/_workitems/edit/8112)
